java.lang.ClassNotFoundException: org.w3c.dom.ElementTraversal

com.ibm.ws.jsfContainer_fat_2.3 bucket and com.ibm.ws.jsf.2.3_fat on Java 8 encounter this ClassNotFoundException after an XML change here: https://github.com/OpenLiberty/open-liberty/pull/18862

```

Caused by: java.lang.ClassNotFoundException: org.w3c.dom.ElementTraversal
        at org.apache.tools.ant.AntClassLoader.findClassInComponents(AntClassLoader.java:1386)
        at org.apache.tools.ant.AntClassLoader.findClass(AntClassLoader.java:1335)
        at org.apache.tools.ant.AntClassLoader.loadClass(AntClassLoader.java:1090)
        at org.apache.tools.ant.util.SplitClassLoader.loadClass(SplitClassLoader.java:61)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 42 more
```
        
Fix is to add the xml api back to the build.gradle.

--- a/dev/com.ibm.ws.jsf.2.3_fat/build.gradle
+++ b/dev/com.ibm.ws.jsf.2.3_fat/build.gradle
@@ -27,6 +27,7 @@ dependencies {
     requiredLibs('org.eclipse.birt.runtime:org.w3c.css.sac:1.3.1.v200903091627') { force = true }
     requiredLibs project(':io.openliberty.org.apache.commons.codec')
     requiredLibs project(':io.openliberty.org.apache.commons.logging')
+    requiredLibs 'xml-apis:xml-apis:1.4.01'
 }
 
 addRequiredLibraries.dependsOn addJakartaTransformer

--- a/dev/com.ibm.ws.jsfContainer_fat_2.3/build.gradle
+++ b/dev/com.ibm.ws.jsfContainer_fat_2.3/build.gradle
@@ -38,6 +38,7 @@ dependencies {
     requiredLibs('org.eclipse.birt.runtime:org.w3c.css.sac:1.3.1.v200903091627'){ force = true }
     requiredLibs project(':io.openliberty.org.apache.commons.codec')
     requiredLibs project(':io.openliberty.org.apache.commons.logging')
+    requiredLibs 'xml-apis:xml-apis:1.4.01'
     
     mojarra 'org.glassfish:javax.faces:2.3.3',
       'javax.faces:javax.faces-api:2.3'

